docs/cellxgene_census_schema.md

```diff
@@ -1,8 +1,8 @@
 # CZ CELLxGENE Discover Census Schema
 
-**Version**: 2.2.0
+**Version**: 2.3.0
```

Comment:
I don't feel strongly about this - but note that the semver explanation in the census schema mentions "new modality" as a major schema change ...

Major: any schema changes that make the Census incompatible with the Census API or SOMA API. Examples:
Column deletion in Census obs
**Addition of new modality**

Reply:
I saw that, but I put the emphasis on **incompatible** as the qualifier. We did this in a way that does not break anything. I read the example as having the assumption that adding a new modality would require a breaking change.

Reply:
That makes sense. Consider this comment resolved then.